config/routes.rb

```diff
@@ -2,4 +2,8 @@
 
 Rails.application.routes.draw do
   root 'static#index'
+
+  resources :rooms, only: [:create]
+  get 'rooms/:room_hash', to: 'rooms#show', as: 'room'
```

Comment:
we wouldn't need extra routes, just plain `resources` actions if we define `to_param`